operating_unit/models/operating_unit.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2015-TODAY Eficent
+# - Jordi Ballester Alomar
+# Copyright 2015-TODAY Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from odoo import api, fields, models
+
+
+class OperatingUnit(models.Model):
+
+    _name = "operating.unit"
+    _description = "Operating Unit"
+
+    name = fields.Char(required=True)
+    code = fields.Char(required=True)
+    active = fields.Boolean(default=True)
+    company_id = fields.Many2one(
+        "res.company",
+        required=True,
+        readonly=True,
+        default=lambda self: self.env.company,
+    )
+    partner_id = fields.Many2one("res.partner", "Partner", required=True)
+    user_ids = fields.Many2many(
+        "res.users", "operating_unit_users_rel", "poid", "user_id", "Users Allowed"
```

Comment:
I think version 13 is a good moment to put a good column name here. It should be something like `operating_unit_id` or `ou_id`. Migration script could be added later, after the merge.

Reply:
Thanks for your review! I already rename the column to operating_unit_id